src/scss/base.scss

```diff
@@ -27,37 +29,102 @@
 		@include oFormsCommonSmall;
 	}
 
-	@include oFormsRadioCheckbox();
+	/// Radio and checkboxes
+
+	.#{$class}__checkboxes,
+	.#{$class}__radios {
+		@include oFormsRadioCheckboxContainer();
+	}
+
+	.#{$class}__radio,
```

Comment:
I would argue that there is potential for this to raise issues, the difference between `radio` and `radios` may be too subtle, perhaps we may need to change the element/modifier situation, or the name of the wrapper?

Reply:
and also could we not refer to the wrapper of radios and checkboxes as something that wraps 'options'? Or 'choices' or 'selection' or something like that?